zipkin/src/test/java/zipkin/internal/Span2Test.java

```diff
@@ -119,19 +146,17 @@
     );
   }
 
+  /** Test serializable as used in spark jobs. Careful to include all non-standard fields */
   @Test public void serialization() throws Exception {
     Buffer buffer = new Buffer();
-    new ObjectOutputStream(buffer.outputStream()).writeObject(base);
 
-    assertThat(new ObjectInputStream(buffer.inputStream()).readObject())
-      .isEqualTo(base);
-  }
+    Span2 span = base.toBuilder()
+      .addAnnotation(1L, "foo")
+      .build();
 
-  @Test public void serializationUsesJson() throws Exception {
```

Comment:
removed requirement to use json as serialization as it isn't important and adds code. people can use whatever they want, like kryo